app/assets/locales/android_translatable_strings.txt

```diff
@@ -428,6 +428,7 @@ no.file.browser.title=No File Browser Available
 no.file.browser=Your device does not have a file browser installed. Please download one from the playstore and then try again.
 template.error=Your template is not correctly formatted
 
+file.selection.invalid.extension=CommCare cannot select this file as it doesn't have an extension. Please rename the file to include a valid extension.

```

Comment:
It will be quite hard for most our users to know the valid extension of a file. I think we should just show the error without the directive to rename the file.